src/app/modules/submission/email-submission/email-submission.util.ts

```diff
@@ -0,0 +1,160 @@
+import { BasicField } from '../../../../types'
+import {
+  ProcessedCheckboxResponse,
+  ProcessedTableResponse,
+} from '../submission.types'
+
+import {
+  EmailAutoReplyField,
+  EmailDataForOneField,
+  EmailJsonField,
+  ResponseFormattedForEmail,
+} from './email-submission.types'
+
+/**
+ * Determines the prefix for a question based on whether it is verified
+ * by MyInfo.
+ * @param {Object} response
```

Comment:
nit: JSDocs in TS (should they be called TSDOcs????!?!) don't need typings anymore, double typing may desync/confuse

Reply:
yeah sorry, copied these over from the old functions and missed out deleting the types